core/src/main/groovy/noe/tomcat/configure/ConnectorAttributesTransformer.groovy

```diff
@@ -178,8 +179,19 @@ class ConnectorAttributesTransformer {
         node.appendNode("UpgradeProtocol", ['className': connector.getUpgradeProtocol().getClassName()])
       }
 
+      if (connector.getSSLHostConfig() != null) {
```

Comment:
It seems to me, that Groovy has "build-in" good tools for manipulation w/ XMLs. Do we really need a new dependency?
Wouldn't smth similar to ` node.appendNode("SSLHostConfig", connector.getSSLHostConfig() as Map)`  work?

However, my biggest concern would - with this "shortcut" we would lose the very basic validation -isn't it dangerous? Will it not generate unexpected data?
Additionally, it is a bit different concept for 1 element only - consistency is what makes code easy to understand quickly.

Reply:
WDYT?

Reply:
Did not forget it :) working on this